packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -47,23 +47,23 @@ const ESGlobalKeys = new Set([
 
     // *** 19.3 Constructor Properties of the Global Object
     'AggregateError',
-    'Array',
```

Comment:
Ok, the changes here are that any typed array, it should be remapped. It qualifies as category C from the comment above this block. The side effect here is that now operations on those typed arrays are going to be slower, which is a problem since those arrays are meant to provide down to the metal operation.

Reply:
Additionally, I think that anything that is part of the `ReflectiveIntrinsicObjectNames` should not be here, it is an unnecessary duplication, so we should probably clean this up.

Reply:
Done